### PR TITLE
JS domain style customisation

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

File: static/css/js.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.js.function {
+    padding: 16px;
+    border-radius: 4px;
+    background-color: #F9FAFB;
+    border: 2px solid #00A09D;
```
Comment (edi-odoo) [inline static/css/js.css:5]: `border: 2px solid #017e84;`

File: static/css/js.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.js.function {
+    padding: 16px;
```
Comment (edi-odoo) [inline static/css/js.css:2]: `1rem`
(prefer `rem` values)

File: static/css/js.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.js.function .sig-param, .js.function .sig-paren {
+    font-style: normal;
+    font-size: initial;
+    color: #4B5563;
+    font-family: var(--bs-font-monospace);
+}
+
+.js.function .sig-paren {
+    color: black;
```
Comment (edi-odoo) [inline static/css/js.css:42]: `color: #212529`

File: static/css/js.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.js.function {
+    padding: 16px;
+    border-radius: 4px;
```
Comment (edi-odoo) [inline static/css/js.css:3]: `border-radius: .25rem;`

**Review by edi-odoo — CHANGES_REQUESTED** [review]
We could later place this inside style.scss

**Comment by SimonGenin** [comment]
@Feyensv Changes applied :)

**Comment by Feyensv** [comment]
@robodoo delegate+

**Comment by SimonGenin** [comment]
@robodoo r+

**Pull request closed without merging**